packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -835,6 +835,7 @@ private void addSpansFromStyleAttributes(SpannableStringBuilder workingText) {
               mFontWeight,
               getFontFeatureSettings(),
               mFontFamily,
+              null,
```

Comment:
Do we need to add support for text input as well?

Reply:
The intent here, is to add a span over the full set of content, for attributes set on the EditText instead of underlying Spannable content.

It looks like line 766 here would need to be updated, since right now, it will strip any underlying CustomStyleSpan set if its other properties match those set on the full EditText. So, we should set `fontVariationSetting` on underlying EditText, similar to `fontFeatureSettings`.